### PR TITLE
Responsive tile layout

### PR DESCRIPTION
**Pull request by devin5**
this PR reverts to V3 styles. Then add styling and markup refactors to change the entire app to be a fixed screen height at whatever your current screen height is - the height of Lookers header. This will eventually be on a tv with so no scroll was needed.

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -22,9 +22,15 @@ import "./styles/animationStyle.css";
 const useStyles = makeStyles(() => ({
   appContainer: {
     padding: "10px 15px",
+    minWidth: " 100%",
+    minHeight: " 100%",
```
Comment (danchapman) [inline src/App.jsx:26]: Any reason in particular you are using min and max here? If you are using a percentage, width and height will work.

File: src/GridController.jsx [hunk 2]
```diff
@@ -1,13 +1,22 @@
...
 import TileGrid from "./TileGrid.jsx";
-
+const useStyles = makeStyles(() => {
+  gridController: {
+    minWidth: "100%";
+    minHeight: "100%";
+    height: "100%";
+    width: "100%";
+    maxHeight: "100%";
+    maxWidth: "100%";
```
Comment (danchapman) [inline src/GridController.jsx:11]: again, all 6 of these are the same as writing `width: 100%` and `height: 100%`

File: src/Header.jsx [hunk 3]
```diff
@@ -37,13 +37,13 @@ const useStyles = makeStyles(() => ({
 
   headerGridStart: {
     opacity: 0,
-    height: "5vh",
-    minHeight: "45px",
+    height: "35px",
+    minHeight: "35px",
```
Comment (danchapman) [inline src/Header.jsx:41]: this is also the same as just `height: "35px"
`

File: src/OverlayVis.jsx [hunk 4]
```diff
@@ -19,13 +15,17 @@ const useStyles = makeStyles(() => ({
     zIndex: 100,
     width: "100%",
     height: "100%",
+    maxWidth: "100%",
+    MaxHeight: "100%",
```
Comment (danchapman) [inline src/OverlayVis.jsx:19]: same, unnecessary

File: src/OverlayVis.jsx [hunk 5]
```diff
@@ -19,13 +15,17 @@ const useStyles = makeStyles(() => ({
...
     pointerEvents: "none",
     overflow: "hidden",
   },
   radial: {
     zIndex: 100,
     width: "100%",
     height: "100%",
+    maxWidth: "100%",
+    MaxHeight: "100%",
```
Comment (danchapman) [inline src/OverlayVis.jsx:28]: Unnecessary

File: src/OverlayVis.jsx [hunk 6]
```diff
@@ -34,9 +34,11 @@ const useStyles = makeStyles(() => ({
   overlayBackground: {
     backgroundColor: "white",
     zIndex: 50,
-    width: "100%",
+    width: "80%",
     height: "100%",
-    marginLeft: 180,
+    maxWidth: "100%",
+    MaxHeight: "100%",
```
Comment (danchapman) [inline src/OverlayVis.jsx:40]: mixing width maxWidth with percentages again is incorrect

File: src/RadialTile.jsx [hunk 7]
```diff
@@ -1,38 +1,37 @@
...
 
-const useStyles = makeStyles(() => ({
+const useStyles = makeStyles((theme) => ({
   tileStart: {
     opacity: 0,
-    height: "45vh",
-    padding: "0.5vh 10px",
-    minHeight: "410px",
+    height: "460px",
+    maxHeight: "460px",
```
Comment (danchapman) [inline src/RadialTile.jsx:16]: here too

File: src/RadialTile.jsx [hunk 8]
```diff
@@ -1,38 +1,37 @@
...
+    },
   },
   tileEnd: {
     opacity: 1,
     transition: ".4s",
-    height: "45vh",
-    padding: "0.5vh 10px 0 10px",
-    minHeight: "410px",
+    padding: "0 10px 0 10px",
+    // maxHeight: "460px",
```
Comment (danchapman) [inline src/RadialTile.jsx:26]: delete

File: src/RadialTile.jsx [hunk 9]
```diff
@@ -1,38 +1,37 @@
...
+    marginTop: "5px",
   },
   radialContainer: {
-    width: "100%",
-    height: "340px",
-    overflow: "hidden",
+    maxHeight: "86%",
+    height: "86%",
```
Comment (danchapman) [inline src/RadialTile.jsx:31]: here too

File: src/styles/animationStyle.css [hunk 10]
```diff
@@ -56,23 +56,25 @@
   transition: all 0.2s linear;
 } */
 
+body {
+  min-height: "100%";
```
Comment (danchapman) [inline src/styles/animationStyle.css:60]: do not use % values for min or mx

File: src/App.jsx (hunk 1 above)
Comment (devin5) [reply src/App.jsx:26]: I am going thrpugh every one and deleting in order from child to parent

Comment (danchapman) [reply src/App.jsx:26]: change those two to `height: 100%`

**Review by danchapman — APPROVED** [review]
🍰

**Pull request merged**